packages/effects/src/_variables.scss

```
@@ -1,12 +1,10 @@
 :root {
   // Shadows
-  --dialog-strong: 0px 16px 48px rgb(0 0 0 / 16%);
-  --dialog:
-    0px 0px 0px 0.5px rgb(0 0 0 / 6%), 0px 12px 24px rgb(0 0 0 / 8%),
-    0px 2px 3px rgb(0 0 0 / 4%);
-  --content: 0px 2px 3px rgb(0 0 0 / 6%);
-  --canvas: 0px 0px 0px rgb(0 0 0 / 0%);
-  --keyboard-key: 0 2px 0px var(--grey4);
+  --dialog-strong: 0px 16px 48px rgba(0, 0, 0, 0.16);
```

Stylelint (17.9.0)

[error] Expected "0.16" to be "16%" (alpha-value-notation)

(alpha-value-notation)